zebra-consensus/src/block/subsidy/funding_streams/tests.rs

```diff
@@ -11,13 +11,13 @@ fn test_funding_stream_values() -> Result<(), Report> {
     let network = Network::Mainnet;
```

Comment:
Optional/Nitpick: We could also update the value of `network` to minimize changes.

```suggestion
    let network = &Network::Mainnet;
```

Reply:
change applied